core/domain/html_cleaner.py

```diff
@@ -255,8 +310,8 @@ def convert_to_text_angular(html_data):
                     link = soup.new_tag('oppia-noninteractive-link')
                     url = tag['href']
                     text = tag.get_text()
-                    link['url-with-value'] = url
-                    link['text-with-value'] = text
+                    link['url-with-value'] = '&quot;' + url + '&quot;'
```

Comment:
This should be json.dumps() / escape() or something right? Similarly to the issues with tabs/collapsibles.

Make sure that the creation process for these attr values is exactly the same here as it currently is in the client.

Reply:
Done!